SwiftReflector/TypeMapping/TypeDatabase.cs

```diff
@@ -305,6 +316,13 @@ void ReadVersion1_0 (XElement typeDbRoot, ErrorHandling errors)
 				foreach (var op in ops) {
 					AddOperator (op);
 				}
+
+				var aliases = from alias in entityList.Elements ("typealias")
+					      select TypeAliasDeclaration.FromXElement (null, alias);
+
+				foreach (var alias in aliases) {
+					AddTypeAlias (alias);
+				}
```

Comment:
No need to change, could have been in the LINQ with the ForEach call.

Reply:
Or I could have put in an AddRange type call...